apps/web/components/eventtype/EventTeamTab.tsx

```diff
@@ -472,10 +472,12 @@ const Hosts = ({
       initialValue.current = { hosts: getValues("hosts"), schedulingType, submitCount };
       return;
     }
-    resetField("hosts", {
```

Comment:
hosts weren't reset when changing the scheduling type as it wasn't set dirty

Reply:
Thank you 🙌 🚀